aten/src/ATen/core/op_registration/kernel_function_test.cpp

```diff
@@ -30,21 +31,23 @@ C10_DEFINE_TENSOR_TYPE(TensorType1);
 C10_DECLARE_TENSOR_TYPE(TensorType2);
 C10_DEFINE_TENSOR_TYPE(TensorType2);
 
-void errorKernel(const Tensor&) {
+int64_t errorKernel(const Tensor& tensor, int64_t input) {
```

Comment:
Why did this change?

Reply:
errorKernel is used in these test cases together with a schema that says the kernel is supposed to return IntType. This was actually a bug but not noticed since nothing checked the schema against the kernel function. Now with this diff, the schema is checked against the kernel function and this threw an error.